addons/mail/static/src/core/common/thread_model.js

```diff
@@ -145,11 +108,17 @@ export class Thread extends Record {
     model;
     /** @type {boolean} */
     areAttachmentsLoaded = false;
-    attachments = Record.many("Attachment");
+    attachments = Record.many("Attachment", { sort: (a1, a2) => (a1.id < a2.id ? 1 : -1) });
     activeRtcSession = Record.one("RtcSession");
     /** @type {object|undefined} */
     channel;
-    channelMembers = Record.many("ChannelMember", { onDelete: (r) => r.delete() });
+    channelMembers = Record.many("ChannelMember", {
+        onDelete: (r) => r.delete(),
+        /** @this {import("models").Thread} */
+        onUpdate() {
+            this._store.updateBusSubscription();
+        },
+    });
```

Comment:
It's annoying that channel member is controlled through this relation rather than managing itself.

Reply:
`ChannelMember` needs `thread` to exist if it was inserted in `ChannelMember` immediately. As `thread` is the inverse field, it will trigger this `onUpdate`. So it's all good